Marco/Day03/hackerrank/solutions/Third.java

```diff
@@ -0,0 +1,25 @@
+package hackerrank.solutions;
+
+import java.util.Scanner;
+
+public class Third {
+
+    public static void solving() {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("================================");
+
+        for(int i = 0; i < 3; i++)
```

Comment:
Even when all programming books teach us for loops with var names like i, j, x, y, etc. please avoid these. All the time try to name your variables as descriptive as possible